src/main/java/com/sintern/api/OpenInternPositionController.java

```diff
@@ -83,4 +85,11 @@ public OpenInternPositionController(OpenInternPositionService openInternPosition
         return openInternPositionService.findOpenInternPositionDTOByCompany(company);
 
     }
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.OK)
+    public void createOpenInternPosition(@RequestBody CreateOpenInternPositionRequest createOpenInternPositionRequest) {
+        openInternPositionService.addOpenInternPosition(createOpenInternPositionRequest.getAvailablePositions(),
+                createOpenInternPositionRequest.getDepartment(), createOpenInternPositionRequest.getDescription(), createOpenInternPositionRequest.getName(), companyService.findByID(createOpenInternPositionRequest.getCompanyID()));
```

Comment:
`companyService.findByID(createOpenInternPositionRequest.getCompanyID())` this should be used in the `OpenInternPositionServiceImpl`. Thus, pass as arguments only the **transformed object** and the `CompanyId`.

Reply:
Done